src/types/UiaTextRangeBase.hpp

```diff
@@ -125,7 +125,7 @@ namespace Microsoft::Console::Types
 
         IRawElementProviderSimple* _pProvider{ nullptr };
 
-        std::wstring _wordDelimiters{};
+        std::wstring _wordDelimiters{ L"" };

```

Comment:
Oh and I'd suggest not doing this, if possible. 😅
That constructor is not `constexpr`!